Funcky.Money/MoneyEvaluationContext.cs

```diff
@@ -104,13 +99,9 @@ private Option<bool> CompatibleRounding()
 
         private MoneyEvaluationContext CreateContext()
             => new(
-                _targetCurrency.GetOrElse(()
-                    => throw new InvalidMoneyEvaluationContextBuilderException("Money evaluation context has no target currency set.")),
+                _targetCurrency.GetOrElse(() => throw new InvalidMoneyEvaluationContextBuilderException("Money evaluation context has no target currency set.")),
```

Comment:
(no need for a trailing dot)

Reply:
A sentence needs a full stop, it's also done like this in System.*

Reply:
That's true, but:

We don't usually do that (in fact we kind of have a rule internally to not end exceptions with dots), and you didn't do it yourself everywhere, either:

![image](https://github.com/polyadic/funcky-money/assets/10061519/7407208d-2f1c-44ea-ba6b-a6800cf46191)

I'm fine with either, just noticed the inconsistency. You can resolve this yourself if you want!

Reply:
I fixed the missing full stop instead.